packages/storybook/src/stories/Accordion.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import { Accordion, AccordionItem } from '@littlespoon/accordion/src/Accordion'
+import { AccordionDetails } from '@littlespoon/accordion/src/AccordionContent'
+import { AccordionSummary } from '@littlespoon/accordion/src/AccordionSummary'
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+
+export default {
+  title: 'Design System/Accordion',
+  component: Accordion,
+} as ComponentMeta<typeof Accordion>
+
+const Template: ComponentStory<typeof Accordion> = (args) => <Accordion {...args} />
+
+export const Default = Template.bind({})
+Default.args = {
+  children: (
+    <AccordionItem>
+      <AccordionSummary>Heading</AccordionSummary>
+      <AccordionDetails>Content</AccordionDetails>
+    </AccordionItem>
+  ),
+}
+
+export const SingleExpanded = Template.bind({})
+SingleExpanded.args = {
+  children: (
+    <AccordionItem>
+      <AccordionSummary isExpanded={true}>Heading</AccordionSummary>
+      <AccordionDetails isExpanded={true}>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+        labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+        laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+        voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
+        cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum
+      </AccordionDetails>
+    </AccordionItem>
+  ),
+}
+
+export const Multiple = Template.bind({})
+Multiple.args = {
+  children: (
+    <>
+      <AccordionItem>
```

Comment:
Since we modularized this to support customization of individual components, we lost the ability to track the logic of which item is currently expanded.

I think we should also export a hook as part of the component elements, something like this:

```
useAccordionController() {
  const [activeItem, setActiveItem] = React.useState<string | null>(null)

  const handleItemClick = (itemName: string) =>  {
      setActiveItem(activeItem === itemName ? null ? itemName)
   }

 return { activeItem, handleItemClick }
}
```

It could be with names or maybe even better with just indexes

Once we have that, we could add a controllable Accordion to the stories